shiny/templates/chat/llm-enterprise/aws-bedrock-anthropic/requirements.txt

```diff
@@ -1,5 +1,4 @@
 shiny
 python-dotenv
-tokenizers
 chatlas
 anthropic[bedrock]
```

Comment:
Just a note that reading through this PR reminded me of an issue I opened in chatlas to make provider-specific extras in `chatlas`. Here's the related PR: https://github.com/posit-dev/chatlas/pull/66

In the future (I guess not yet...) it'd be nice to be able to include those extras in these templates -- e.g. `chatlas[bedrock-anthropic]` -- which would mean that updates to the provider dependencies don't require changes here to stay up-to-date.

Reply:
Yea, agreed, thanks for the PR

FWIW, `tokenizers` was once for `ui.Chat.messages(token_limits)` itself, but we longer advertise it